src/Fantomas/Context.fs

```diff
@@ -749,6 +749,12 @@ let private findTriviaMainNodeOrTokenOnEndFromRange nodes (range:range) =
     |> List.tryFind(fun n ->
         Trivia.isMainNode n && RangeHelpers.rangeEq n.Range range
         || Trivia.isToken n && RangeHelpers.rangeEndEq n.Range range)
+    
+let private findTriviaMainNodeOrTokenContainedInRange nodes (range:range) =
```

Comment:
I would move this function to TriviaHelpers.fs

Reply:
Ah, I thought it should be added here due to `Context.fs` already having `findTriviaMainNodeOrTokenOnStartFromRange` and `findTriviaMainNodeFromRange` functions. But I can move it to `TriviaHelpers.fs`. Would you like it to be renamed?

Reply:
Yeah TriviaHelpers came after we already have some stuff in Context.fs
Not everything was perfectly moved to TriviaHelpers.
Please move and rename your function, we can move the other functions in a cleanup PR.

[TriviaNodeType ](https://github.com/fsprojects/fantomas/blob/master/src/Fantomas/TriviaTypes.fs#L49 )can only be two things so the `Trivia.isMainNode` and `Trivia.isToken` can also be left out I think.

Reply:
Alright, done! I've renamed the function to `findInRange` to better fit with the existing `findByRange` function in `TriviaHelpers.fs`. Let me know if that is okay or if you' prefer a different name.